### DISCUSSION TITLE
Beginner Help Modifying Glow

### DISCUSSION THREAD

**Discussion by Independence-hour7** [klaasnicolaas/home-assistant-glow#417] [General]
So long story short, first time ever using an ESP board, now i have installed HA Glow using the step by step guide and all seems to be working well, i would simply like to reduce the amount of data as there seem to be multiple updates a second.

My problem is now that it is in HA i've got no idea where to modify the code as listed below

https://glow-energy.io/docs/faq/reduce_sensor_data

i tried inside ESPHome add on -> Edit but theres only about 15 lines or so of code in there so that doesnt seem right. What am i missing here?

**Comment by klaasnicolaas** [comment]
Please see also #409
